### ISSUE TITLE
IDEA plugin - jsTestDriver

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#280]
```
I'm using the IDEA plugin jsTestDriver 1.3.8 in IDEA 10.5.2

What steps will reproduce the problem?
1. I create a js test i.e: myJsTest.js. Create a simple test case.
2. right click on the fie and select "Run myJsTest.js" 


What is the expected output? What do you see instead?
My expected output would be that the test to be automatically run or at least 
popup the test config window to specify the test configuration file. But the 
IDEA is hanging, it is not responding to anything. the screen is freeze. I have 
to kill the process.

In case I'm doing the same steps with the *.conf file it's works fine. i.e: 
right click on the JsTestDriver.conf -> run, works.

What version of the product are you using? On what operating system?
 jsTestDriver 1.3.8 in IDEA 10.5.2

Please provide any additional information below.

Also noticed that If I chose "Create "myTest.js" will popup a configuration 
window where I can specify the configuration file etc. It is really does. But 
if I run that configuration the IDEA is hanging I have to kill the process to 
be responsive.

I will try to attach a patch for this.. hope I will find some time these days.

```

Original issue reported on code.google.com by `alpar....@gmail.com` on 11 Oct 2011 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
Oh, yes, you are right.
Configuration window is a good idea.
```

Original comment by `Sergey.S...@gmail.com` on 11 Oct 2011 at 4:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 13 Oct 2011 at 10:54
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Yes this is also a problem for us. Wanting to run single tests/test cases 
rather than the entire suite is a massive productivity gain. I can run a these 
with the 3 step process of:
1. Right click a test/test case choose "create"
2. Select a "custom" conf file (using the "generated" option hangs intellij)
3. Now run the configuration it just created.

The annoying thing is that step 2 doesnt adhere to defaults. If i do the 
following:
1. Run > Edit Configurations
2. Expand the Defaults section and select JSTestDriver
3. From the "Test" drop down select "JavaScript Test File"
4. Set the "JSTD Configurations file" to "Custom" and specify the location of 
the conf file
5. Click "OK"

That should default every new run configuration to have the custom conf file as 
specified and at least make running individual tests a 2 step process instead 
of 3.

If the defaults issue is fixed then you could really make this a one step 
process by when you click "run" if a config doesnt exist, create it using 
defaults, and then run it automatically.


Hope this information is useful.
Michael
```

Original comment by `michael....@gmail.com` on 12 Feb 2012 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
We ran into the same problem and unfortunately it's a dealbreaker for us when 
it comes to using this plugin. Running the tests works great with our 
predefined config file but as soon as we use "Run context configuration" to run 
a singe test method or a test case, the config file we set up in the default 
configuration is ignored and a new one is generated. The generation process 
takes about a minute and leaves the IDE unresponsive for that time.

Would it be possible to add some kind of default settings for the context 
configuration?
```

Original comment by `zoltan.o...@gmail.com` on 27 Apr 2012 at 12:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed in WebStorm 5.0, PhpStorm 5.0, IntelliJ IDEA 12, PyCharm 2.6.
```

Original comment by `Sergey.S...@gmail.com` on 7 Sep 2012 at 7:11
- Changed state: **Fixed**
